src/styles/style.css

```diff
@@ -0,0 +1,91 @@
+/* Eric Meyer's CSS Reset */
+html, body, div, span, applet, object, iframe,
```

Comment:
You don't need this Reset when you have your own `reset.css`

Reply:
Removed Eric Meyer's CSS Reset